sdk/dotnet/GetSshKey.cs

```diff
@@ -34,18 +34,15 @@ public GetSshKeyArgs()
     public sealed class GetSshKeyResult
     {
         public readonly string Fingerprint;
-        /// <summary>
-        /// The provider-assigned unique ID for this managed resource.
-        /// </summary>
-        public readonly string Id;
+        public readonly int Id;
```

Comment:
That's a breaking change for us

Reply:
This is a breaking change BUT this is the correct type of the parameter.